README.md

```diff
@@ -194,9 +194,9 @@ To connect your application to Unleash you'll need to use a client SDK for your
 - [Rust SDK](https://github.com/unleash/unleash-client-rust)
 - [.NET SDK](https://docs.getunleash.io/reference/sdks/dotnet)
 
-**Official front-end SDKs:**
+**Official frontend SDKs:**
 
-The front-end SDKs connect via [Unleash Edge](https://docs.getunleash.io/reference/unleash-edge) in order to ensure privacy, scalability and security.
+The frontend SDKs connect via [Unleash Edge](https://docs.getunleash.io/reference/unleash-edge) in order to ensure privacy, scalability and security.
```

Comment:
Out of scope for this PR but this sounds like frontend SDKs can only connect to Edge.

Reply:
agreed, I'll take this in my follow-up PR